AutomatedTesting/Gem/PythonCoverage/Code/Source/PythonCoverageEditorSystemComponent.h

```diff
@@ -74,11 +74,15 @@ namespace PythonCoverage
         void WriteCoverageFile();
 
         CoverageState m_coverageState = CoverageState::Disabled; //!< Current coverage state.
-        AZStd::unordered_map<AZStd::string, AZStd::unordered_map<AZ::Uuid, AZ::ComponentDescriptor*>> m_entityComponentMap; //!< Map of
-        //!< component IDs to component descriptors for all activated entities, organized by test cases.
+        AZStd::unordered_map<AZ::Uuid, AZ::ComponentDescriptor*> m_entityComponents; //!< Set of
+        //!< component IDs to component descriptors for all activated entities.
```

Comment:
minor: Despite the comment change, this still appears to be a map, not a set. Still should be a nice improvement to store (and compare!) the UUID instead of its string version.

Reply:
Fixed 👍